checador/spec/helpers/records_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the RecordsHelper. For example:
+#
+# describe RecordsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe RecordsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Yoy may remove this file

Reply:
Okay!
I'm going to delete it in the next PR!